src/basic_application.rs

```diff
@@ -0,0 +1,51 @@
+//! # Basic Application Sample
+//!
+//! This sample demonstrates how to create a GTK application with a toplevel `window`, set its title, size and position, how to add a `button` to this `window` and how to connect signals with actions.
+
+#![crate_type = "bin"]
+
+extern crate gio;
+extern crate gtk;
+
+use gtk::prelude::*;
+
+const APP_ID: &'static str = "org.gtk-rs.basic_app";
+const APP_FLAGS: gio::ApplicationFlags = gio::APPLICATION_FLAGS_NONE;
+
+#[cfg(feature = "gtk_3_10")]
```

Comment:
In https://github.com/gtk-rs/gtk/blob/master/src/application.rs option `application_id` start from v3_6.
https://developer.gnome.org/gtk3/stable/GtkApplication.html#gtk-application-new says same